src/metatensor/models/cli/eval.py

```diff
@@ -21,7 +21,11 @@
 
 
 logger = logging.getLogger(__name__)
-logging.basicConfig(level=logging.INFO, format="%(message)s")
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(asctime)s][%(levelname)s] - %(message)s",
```

Comment:
I think giving the time here is not necessary. But if you want it we can add it.